### ISSUE TITLE
3-hydroxyisovaleric aciduria

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6014]
**Preferred term label:** 3-hydroxyisovaleric aciduria

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of 3-hydroxyisovaleric acid in the urine is above the normal range., PMID: 29844061

**Parent term (use hpo.jax.org/app)** Organic aciduria HP:0001992

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
